lib/infrastructure/class-wp-rest-server.php

```diff
@@ -15,22 +15,85 @@
  * @package WordPress
  */
 class WP_REST_Server {
+		/**
+	 * GET transport method
```

Comment:
Sure because you are right standards-wise, even though most of them are not complete sentences and therefore do not deserve a period.